<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudgetApp</class>
 <widget class="QWidget" name="BudgetApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budget Tracker</string>
  </property>

  <widget class="QComboBox" name="categoryDropdown">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Groceries</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gas</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rent</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Utilities</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Weekly Paycheck</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dining</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Entertainment</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
</widget>

  <widget class="QLineEdit" name="descriptionInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Description</string>
   </property>
  </widget>

  <widget class="QLineEdit" name="amountInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Amount</string>
   </property>
  </widget>

  <widget class="QComboBox" name="typeDropdown">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Income</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Expense</string>
    </property>
   </item>
  </widget>

  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Transaction</string>
   </property>
  </widget>

  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
